versions.yaml

```diff
@@ -290,13 +290,13 @@ externals:
   virtiofsd:
     description: "vhost-user virtio-fs device backend written in Rust"
     url: "https://gitlab.com/virtio-fs/virtiofsd"
-    version: "v1.3.0"
+    version: "v1.5.0"
```

Comment:
There's a PR already opened for virtiofsd bump. Added this change for time being to test as using the Depends-on was causing failure.